test/hardware/TestFlowPrediction/HardwareTestFlowPrediction.c

```diff
@@ -64,17 +72,18 @@ static void loadConfigToFlash() {
 }
 
 static void runTest() {
-    env5HwFpgaPowersOn();
+    env5HwControllerFpgaPowersOn();
 
     uint8_t data[] = {53, 49, 22, 53, 61, 35};
     uint8_t ref_res[] = {38, 63};
 
     for (int counter = 0; counter < 2; counter++) {
         for (int i = 0; i < 3; i++) {
             uint8_t offset = counter * 3;
-            uint8_t return_val = (uint8_t)flow_prediction_predict(data + offset);
+            // TODO fix flow_prediction.h
+            /*uint8_t return_val = (uint8_t)flow_prediction_predict(data + offset);
 
-            PRINT("Ture: %d, Predicted: %d", ref_res[counter], return_val);
+            PRINT("Ture: %d, Predicted: %d", ref_res[counter], return_val);*/
```

Comment:
Commented code is almost always dangerous.
If we had to comment stuff out to make things build or run, i think we have to come up with a different solution. Maybe we can skip these targets and issue a warning about them being broken.